infra/base-images/base-runner/install_perf.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -u
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# try the easiest way first and check if installation succeeded
+apt-get update && apt-get install -y linux-tools-common linux-tools-generic
```

Comment:
Although my knowledege of docker is somewhat limited, my impression is that something like perf will only work if the kernel of the host supports it. Do you know if that's why you need to try multiple times?

Reply:
Indeed.
These ways to try come from https://xiaoyanzhuo.github.io/2019/01/18/Perf-Tool.html